src/garage/tf/algos/batch_polopt.py

```diff
@@ -264,10 +276,20 @@ def init_opt(self):
         """
         raise NotImplementedError
 
-    def get_itr_snapshot(self, itr):
-        """Get all the data that should be saved in this snapshot iteration."""
-        raise NotImplementedError
-
     def optimize_policy(self, itr, samples_data):
-        """Optimize the policy using the samples."""
+        # pylint: disable=no-self-use
```

Comment:
i think all your pylint problems here will go away i you just declare this method abstract (which it is, anyway).